contact.html

```diff
@@ -0,0 +1,51 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+
+  <title>Sprint Challenge - Contact</title>
+
+  <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
+  <link rel="stylesheet" href="css/index.css">
+
+</head>
+<body>
+        <div class="container contact-page">
+                <header>
+                    <div class="nav-container">
+                        <div class="logo">
+                            <img src="img/lambda-black.png" alt="Logo">
+                        </div>
+                        <nav>
+                            <a href="index.html">Home</a>
+                            <a href="about.html">About</a>
+                            <a href="#">Products</a>
+                            <a href="#">Blog</a>
+                            <a href="#">Contact</a>
+                        </nav>
+                    </div>
+                <section class="form">
```

Comment:
This is completely fine, but a class name like `form-section` would probably be a bit more meaningful.